features/promotion/managing_promotions/adding_promotion_with_action_in_different_currencies.feature

```diff
@@ -0,0 +1,25 @@
+@managing_promotions
+Feature: Adding a new promotion with action configured in different currencies
+    In order to give possibility to pay specifically less price for some goods
+    As an Administrator
+    I want to add a new promotion with action configured in different currencies to the registry
+
+    Background:
+        Given the store has currency "USD"
+        And the store has currency "GBP" with exchange rate 0.5
+        And the store operates on a channel named "Web-EU"
+        And that channel allows to shop using "USD" and "GBP" currencies
+        And that channel uses the "USD" currency by default
```

Comment:
Web-EU is using `GBP` and `USD`. In a few weeks/months/years/centuries/millenniums/eons(who knows?) both will be equally typical EU currency ;)